src/utils/playAll.ts

```diff
@@ -1,6 +1,6 @@
-import type { StoryObj } from "@storybook/react";
+import type { Meta, StoryObj } from "@storybook/react";
 
-export function playAll<Story extends StoryObj>(
+export function playAll<Story extends StoryObj<Meta<TCmpOrArgs>>, TCmpOrArgs>(
```

Comment:
@kasperpeulen I wonder if you can help me here. The previous type definition seemed to "throw" away the args type specificity and then complain if you passed a specific story object into it (it would say that `args: {}` isn't assignable to `args: { something... }`)

This seems to fix it but feels like it's not really "correct", IDK?

Reply:
I can imagine that the StoryObj doesn't really work well like a generic constraint, as it is so complex with its conditional types. Maybe it is better to use `StoryAnnotations` which has no conditions:

```ts
import { ReactRenderer } from "@storybook/react";
import { StoryAnnotations } from "@storybook/types";

export function playAll<Story extends StoryAnnotations<ReactRenderer>>(
```

Reply:
That works, great!